playground/pages/blog.vue

```diff
@@ -7,10 +7,11 @@ defineI18nRoute({
   }
 })
 
-const { t, locales, baseUrl } = useI18n({
+const { t, locales } = useI18n({
   useScope: 'local'
 })
-console.log('locales', locales, baseUrl)
+const { url } = useSiteConfig()
```

Comment:
TODO: this should require enabling the `nuxt-site-config` module on the playground.